nuitka/codegen/CodeGeneration.py

```diff
@@ -548,6 +549,7 @@ def makeGlobalContext():
         "EXPRESSION_BUILTIN_REF": generateBuiltinRefCode,
         "EXPRESSION_BUILTIN_EXCEPTION_REF": generateExceptionRefCode,
         "EXPRESSION_BUILTIN_ANONYMOUS_REF": generateBuiltinAnonymousRefCode,
+        "EXPRESSION_BUILTIN_ABS_REF": generateBuiltinAbsCode,
```

Comment:
There should be no `REF`. That part of the name stands for reference, and makes we are having some kind of name lookup. This is not the case here. That probably means you also misnamed the operation class.

Reply:
Ah, I see you did not match the names in the `abs` node and here, if they do not, this will simply not have an effect, and compiling to C should give an error for everything not statically optimized.